src/Hit/ColorTerminal.hs

```diff
@@ -116,18 +117,22 @@ arrow = " ➤  "
 data Answer = Y | N
 
 -- | Parse an answer to 'Answer'
-yesOrNo :: Text -> Maybe Answer
-yesOrNo (T.toLower -> answer )
-    | T.null answer = Just Y
+yesOrNo :: Answer -> Text -> Maybe Answer
+yesOrNo def (T.toLower -> answer )
+    | T.null answer = Just def
     | answer `elem` ["yes", "y", "ys"] = Just Y
     | answer `elem` ["no", "n"]  = Just N
     | otherwise = Nothing
 
-prompt :: IO Answer
-prompt = do
-    answer <- yesOrNo . T.strip <$> getLine
+yesOrNoText :: Answer -> Text
+yesOrNoText N = boldCode <> "[n]" <> resetCode <> "/y"
+yesOrNoText Y = boldCode <> "[y]" <> resetCode <> "/n"
+
+prompt :: Answer -> IO Answer
```

Comment:
Could you please add docs here as well explaining what `Answer` it expects as the input?